shell/src/main/java/alluxio/cli/fs/command/DistributedLoadUtils.java

```diff
@@ -90,6 +90,9 @@ private static void load(AbstractDistributedJobCommand command, List<URIStatus>
     ListStatusPOptions options = ListStatusPOptions.newBuilder().setRecursive(true).build();
     command.mFileSystem.iterateStatus(filePath, options, uriStatus -> {
       if (!uriStatus.isFolder()) {
+        if (!uriStatus.isCompleted()) {
+          return;
```

Comment:
Please give some output here

Reply:
add some logs please

Reply:
+1

Reply:
Attempt load failed because: %s is in incomplete status 
-> 
Ignored load because: %s is in incomplete status